### PR TITLE
chore(analytics toolbox): Upload multiple bands of the same raster in one upload

### PR DESCRIPTION
**Pull request by luipir**
Story details: https://app.shortcut.com/cartoteam/story/288224

### CODE REVIEW

File: raster_loader/cli/bigquery.py [hunk 1]
```diff
@@ -24,11 +24,20 @@ def bigquery(args=None):
 @click.option("--project", help="The name of the Google Cloud project.", required=True)
 @click.option("--dataset", help="The name of the dataset.", required=True)
 @click.option("--table", help="The name of the table.", default=None)
-@click.option("--band", help="Band within raster to upload.", default=1)
+@click.option(
```
Comment (vdelacruzb) [inline raster_loader/cli/bigquery.py:27]: Should we allow repeating or do comma separated?

File: raster_loader/cli/bigquery.py [hunk 2]
```diff
@@ -24,11 +24,20 @@ def bigquery(args=None):
...
+    help="Band(s) within raster to upload. "
+    "Could repeat --band to specify multiple bands.",
+    default=[1],
+    multiple=True,
+)
 @click.option(
     "--band_column_name",
-    help="Column name used to store band (Default: band_<band_num>).",
-    default=None,
+    help="Column name(s) used to store band (Default: band_<band_num>). "
```
Comment (vdelacruzb) [inline raster_loader/cli/bigquery.py:36]: Same here

File: raster_loader/cli/bigquery.py (hunk 1 above)
Comment (luipir) [reply raster_loader/cli/bigquery.py:27]: default way to manage muliple parameter via Click module si repeating --band (AFAIK), I preferred to use native method than introducing a parser

Comment (luipir) [reply raster_loader/cli/bigquery.py:27]: btw can be changed

File: raster_loader/cli/bigquery.py (hunk 2 above)
Comment (luipir) [reply raster_loader/cli/bigquery.py:36]: same as above ;)

File: raster_loader/cli/bigquery.py (hunk 1 above)
Comment (vdelacruzb) [reply raster_loader/cli/bigquery.py:27]: Ok if it's the default way let's stick to it. I'm too used to the comma separated format.

Comment (luipir) [reply raster_loader/cli/bigquery.py:27]: me too, infact I started to implement in this way, but required more times and I decider that solution that allow me to focus on tests and stability.

**Comment by vdelacruzb** [comment]
As discussed today maybe it's worth it to include one example in the doc including multiple bands.

Also one test with multiple bands would be nice.

File: raster_loader/tests/test_io.py [hunk 3]
```diff
@@ -292,6 +292,25 @@ def test_rasterio_to_bigquery_with_quadbin_raster_multiple_custom():
...
+    with pytest.raises(IOError):
+        rasterio_to_bigquery(
+            os.path.join(fixtures_dir, "mosaic_cog.tif"),
+            table_name,
+            BQ_DATASET_ID,
+            BQ_PROJECT_ID,
+            overwrite=True,
+            client=client,
+            bands_info=[(1, "block"), (2, "custom_band_2")],
+        )
```
Comment (luipir) [inline raster_loader/tests/test_io.py:311]: duplicate this code part and changing:
`bands_info=[(1, "block"), (2, "custom_band_2")],`
as 
`bands_info=[(1, "metadata"), (2, "custom_band_2")],`
e.g. check raise exception also for metadata column

File: docs/source/user_guide/cli.rst [hunk 4]
```diff
@@ -51,15 +58,59 @@ project named ``my-gcp-project``, a dataset named ``my-bigquery-dataset``, and a
 named ``my-bigquery-table``. If the table already contains data, this data will be
 overwritten because the ``--overwrite`` flag is set.
 
-You can also use the ``--output_quadbin`` flag to upload the raster to the BigQuery
-table in a quadbin format. To use this option, the input raster must be a
-``GoogleMapsCompatible`` raster. You can make your raster compatible by converting it with
-the following command with GDAL:
+If the the no band is specified, the first band of the raster will be uploaded. If the
```
Comment (luipir) [inline docs/source/user_guide/cli.rst:61]: remove

**Comment by luipir** [comment]
other that the above little comments, LGTM

**Pull request merged**